style.css

```diff
@@ -5,629 +5,700 @@
   font-style: normal;
   font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-
-  /* or 138% */
-
-  /* Neutrals/N800 */
-
-  color: #172b4d;
-  display: flex;
+  font-size: 23px;
+  line-height: 50px;
+  width: 45%;
+  color: #262626;
 }
 
 form {
-  margin-top: 70px;
-  background-color: white;
-  width: 327px;
+  width: 52%;
   display: flex;
   flex-direction: column;
-  align-items: center;
 }
 
-input {
-  width: 327px;
+.input-flex {
+  display: flex;
+  justify-content: space-between;
+}
+
+.input-style {
+  width: 48%;
+  padding: 15px;
   border: none;
   border-bottom: 1px solid #dbd8d7;
-  padding: 14px 0 14px 0;
-  font-family: "Inter", sans-serif;
+  font-family: "Roboto", sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 15px;
   line-height: 24px;
-  padding-left: 15px;
 }
 
 #comments {
-  max-width: 327px;
-  max-height: 114px;
-  min-width: 327px;
-  min-height: 114px;
-  border: none;
-  background-color: #fbf8f7;
-  margin-top: 39px;
-  font-family: "Inter", sans-serif;
+  background: #fbf8f7;
+  height: 125px;
+  font-family: "Roboto", sans-serif;
   font-style: normal;
   font-weight: 400;
   font-size: 15px;
-
-  /* or 160% */
   line-height: 24px;
-  padding-left: 20px;
-  padding-top: 20px;
-  color: #3c3a39;
 }
 
-#submit {
-  text-align: center;
-  margin-top: 24px;
-  color: white;
-}
-
-#btn-touch {
+#in-touch-btn {
+  margin: 36px 0;
+  border: none;
+  background-color: var(--orange);
+  padding: 10px;
   color: white;
-}
-
-.submit-button {
-  width: 124px;
-  text-align: center;
-  margin-bottom: 36px;
-  font-family: "Inter", sans-serif;
+  width: 120px;
+  font-family: "Roboto", sans-serif;
   font-style: normal;
   font-weight: 700;
   font-size: 17px;
   line-height: 24px;
+}
 
-  /* identical to box height, or 141% */
-  letter-spacing: 0.03em;
+#in-touch-btn:active {
   background: var(--orange);
-  border: none;
-  padding: 8px;
+  color: white;
 }
 
-.card-btn:hover {
+#in-touch-btn:enabled {
+  background: #e05304;
+  color: white;
+}
+
+#in-touch-btn:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
   color: var(--orange);
-  border: 1px solid var(--orange);
 }
 
-#btn-feature:hover {
-  background-color: #fff5e1;
-  border: 1px solid var(--orange);
+#in-touch-btn:disabled {
+  background: #f4f1f0;
+  color: #5b5957;
 }
 
-.see-project:hover {
-  color: var(--orange);
-  background-color: #fff5e1;
+.footer-line {
+  width: 100%;
+  margin-top: 154px;
+  border: 1px solid #505f79;
 }
 
-#resume:hover {
-  color: var(--orange);
-  background-color: #fff5e1;
-  border: 1px solid var(--orange);
+.links {
+  margin-top: 25px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
 }
 
-#submit:hover {
-  color: var(--orange);
-  background-color: #fff5e1;
+/* MEDIA QUERIES */
+@media (min-width: 950px) {
+  .main-content {
+    margin-left: 100px;
+  }
+
+  .main-title {
+    font-size: 65px;
+    line-height: 65px;
+  }
+
```

Comment:
[ Optional Changes ]

- [ ] It would be nice to adjust the spacing on the description text in the screenshot below, the badges, and the  `See Project` button.

Your Design
![image](https://user-images.githubusercontent.com/50941074/160161063-330a769a-f725-4039-aec4-c4255a47ce7d.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/160161236-eac7e51b-0447-44bb-9bcf-01aed358c38f.png)


- You can use this style anywhere in your media query to fix that. 👍 
``` 
.card-tags {
    margin: 24px 0;
  }
```